fedmsg/consumers/ircbot.py

```diff
@@ -307,9 +313,10 @@ def prettify(self, topic, msg, pretty=False, terse=False, short=False):
                     link = fedmsg.meta.msg2link(msg, **self.hub.config)
 
                 if link and short:
-                    dagd = 'http://da.gd/s'
-                    resp = requests.get(dagd, params=dict(url=link))
-                    link = resp.text.strip()
+                    if callable(short):
```

Comment:
Yes, that's the intention.

As a user - I'd like to still use `requests`, but just a different shortener service.